### ISSUE TITLE
pmesh requirement?

### ISSUE THREAD

**Issue by pmcdonal** [cosmodesi/pypower#18]
This might just be me not understanding something, but in order to do
from pypower import BaseMatrix
without pmesh I needed to comment this out:
https://github.com/cosmodesi/pypower/blob/13c8a65b496452684c07d1fe2ec1edf759e5e27b/pypower/__init__.py#L2
?

**Comment by adematti** [comment]
good catch! should be fixed by [a71b3e3](https://github.com/cosmodesi/pypower/commit/a71b3e31e631841bbc2a6453e86dd62ccee9f012)
